opensanctions/crawlers/md_interdictie.py

```diff
@@ -206,5 +204,38 @@ def parse_control(context: Context, text: str):
                 else:
                     members.append(unknown)                
 
+        print()
         print(f"  owners: { owners }")
-        print(f"  members: { members }")
\ No newline at end of file
+        print(f"  members: { members }")
+
+        entities += list(make_ownerships(context, entity, date, owners))
+        #entities += make_members(entity, date, members)
+
+    return entities
+
+
+def make_ownerships(context, company: Entity, date, owners: [str]) -> [Entity]:
+    for name in owners:
+        name = name.strip()
+        match = re.match("^([^\d\.\()]+)(\(?(\d+\.?\d*) ?%\)?)?$", name)
+        if match:
+            print(match.groups())
+            name = match.group(1)
+            person = context.make("Person")
```

Comment:
Maybe this is a use case for `LegalEntity` to describe the fact that we're not 100% sure it's a person or organization?